src/charactersheet/components/markdown-disclaimer.js

```diff
@@ -0,0 +1,10 @@
+import ko from 'knockout';
+
+ko.components.register('markdown-disclaimer', {
```

Comment:
not to be nitpicky, but wouldn't a `disclaimer` be more like the srd copyright blurb we usually put?
this is more like `markdown-info` or something